vhost-device-spi/src/virtio_spi.rs

```diff
@@ -0,0 +1,71 @@
+// VirtIO SPI definitions
```

Comment:
Everything in this file how is it related to virtio-spi?

This looks more like a Linux UAPI binding.

Reply:
( `s/virtio-snd/virtio-spi` ?)

Reply:
In this file, just some macros are defined, most of them are Linux binding. While I also put the virtio spi config space macros here, like `ConfigMode`

Reply:
Sorry, as @epilys suggested, I meant virtio-spi.

What I mean is that everything defined here is not related to the virtio device, but more to the linux interface, so the filename for me is confusing.

Better something like linux-spi or linux-spi-bindings.

Reply:
Okay